# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# guillermo pacheco <guillopacheco@gmail.com>, 2018
# a1a9b52a3f40dff112eca965c254c602_089360e <d1597d94e208cfb976efe0f2ce18d2e5_734602>, 2018
# Erland Huaman <blackadress.01@gmail.com>, 2021
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-09-28 22:49+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Erland Huaman <blackadress.01@gmail.com>, 2021\n"
"Language-Team: Spanish (Mexico) (https://app.transifex.com/calamares/teams/20061/es_MX/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: es_MX\n"
"Plural-Forms: nplurals=3; plural=n == 1 ? 0 : n != 0 && n % 1000000 == 0 ? 1 : 2;\n"

#: src/modules/bootloader/main.py:46
msgid "Install bootloader."
msgstr "Instalar el cargador de arranque."

#: src/modules/bootloader/main.py:644
msgid "Failed to install grub, no partitions defined in global storage"
msgstr ""

#: src/modules/bootloader/main.py:899
msgid "Bootloader installation error"
msgstr ""

#: src/modules/bootloader/main.py:900
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/displaymanager/main.py:509
msgid "Cannot write LXDM configuration file"
msgstr "No se puede escribir el archivo de configuración de LXDM"

#: src/modules/displaymanager/main.py:510
msgid "LXDM config file {!s} does not exist"
msgstr "El archivo de configuración de LXDM {!s} no existe"

#: src/modules/displaymanager/main.py:598
msgid "Cannot write LightDM configuration file"
msgstr "No se puede escribir el archivo de configuración de LightDM"

#: src/modules/displaymanager/main.py:599
msgid "LightDM config file {!s} does not exist"
msgstr "El archivo de configuración de LightDM {!s} no existe"

#: src/modules/displaymanager/main.py:684
msgid "Cannot configure LightDM"
msgstr "No se puede configurar LightDM"

#: src/modules/displaymanager/main.py:685
msgid "No LightDM greeter installed."
msgstr "LightDM greeter no está instalado."

#: src/modules/displaymanager/main.py:716
msgid "Cannot write SLIM configuration file"
msgstr "No se puede escribir el archivo de configuración de SLIM"

#: src/modules/displaymanager/main.py:717
msgid "SLIM config file {!s} does not exist"
msgstr ""

#: src/modules/displaymanager/main.py:935
msgid "No display managers selected for the displaymanager module."
msgstr "No se seleccionaron gestores para el módulo de gestor de pantalla."

#: src/modules/displaymanager/main.py:936
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"La lista de gestores de pantalla está vacía o indefinida tanto en el "
"globalstorage como en el displaymanager.conf."

#: src/modules/displaymanager/main.py:1023
msgid "Display manager configuration was incomplete"
msgstr "La configuración del gestor de pantalla estaba incompleta"

#: src/modules/dracut/main.py:29
msgid "Creating initramfs with dracut."
msgstr "Creando initramfs con dracut"

#: src/modules/dracut/main.py:63
msgid "Failed to run dracut"
msgstr ""

#: src/modules/dracut/main.py:64
#, python-brace-format
msgid "Dracut failed to run on the target with return code: {return_code}"
msgstr ""

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Trabajo python ficticio."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "Paso python ficticio {}"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "Escribiento fstab."

#: src/modules/fstab/main.py:377 src/modules/fstab/main.py:383
#: src/modules/fstab/main.py:411 src/modules/initcpiocfg/main.py:256
#: src/modules/initcpiocfg/main.py:260 src/modules/initramfscfg/main.py:85
#: src/modules/initramfscfg/main.py:89 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:329 src/modules/networkcfg/main.py:105
#: src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Error de configuración"

#: src/modules/fstab/main.py:378 src/modules/initramfscfg/main.py:86
#: src/modules/mount/main.py:330 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "No hay particiones definidas para que <pre>{!s}</pre> use."

#: src/modules/fstab/main.py:384 src/modules/initramfscfg/main.py:90
#: src/modules/localecfg/main.py:141 src/modules/networkcfg/main.py:106
#: src/modules/openrcdmcryptcfg/main.py:77
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/fstab/main.py:412
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""

#: src/modules/grubcfg/main.py:29
msgid "Configure GRUB."
msgstr "Configura GRUB."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Configurando el reloj del hardware."

#: src/modules/initcpiocfg/main.py:27
msgid "Configuring mkinitcpio."
msgstr "Configurando mkinitcpio"

#: src/modules/initcpiocfg/main.py:257
msgid "No partitions are defined for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initcpiocfg/main.py:261
msgid "No root mount point for <pre>initcpiocfg</pre>."
msgstr ""

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "Configurando initramfs."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Configurando locales."

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "Creando initramfs con mkinitfs."

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Se falló al intentar correr mkinitfs en el objetivo"

#: src/modules/mkinitfs/main.py:50
msgid "The exit code was {}"
msgstr "El código de salida fue {}"

#: src/modules/mount/main.py:43
msgid "Mounting partitions."
msgstr "Montando particiones."

#: src/modules/mount/main.py:164 src/modules/mount/main.py:200
msgid "Internal error mounting zfs datasets"
msgstr ""

#: src/modules/mount/main.py:176
msgid "Failed to import zpool"
msgstr ""

#: src/modules/mount/main.py:192
msgid "Failed to unlock zpool"
msgstr ""

#: src/modules/mount/main.py:209 src/modules/mount/main.py:214
msgid "Failed to set zfs mountpoint"
msgstr ""

#: src/modules/mount/main.py:365
msgid "zfs mounting error"
msgstr ""

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Guardando configuración de red."

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "Configurando el servicio OpenRc dmcrypt."

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Instalar paquetes."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "Procesando paquetes (%(count)d/%(total)d)"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Instalando un paquete."
msgstr[1] "Instalando%(num)d paquetes."
msgstr[2] "Instalando%(num)d paquetes."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Removiendo un paquete."
msgstr[1] "Removiendo %(num)dpaquetes."
msgstr[2] "Removiendo %(num)dpaquetes."

#: src/modules/packages/main.py:740 src/modules/packages/main.py:752
#: src/modules/packages/main.py:780
msgid "Package Manager error"
msgstr ""

#: src/modules/packages/main.py:741
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:753
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""

#: src/modules/packages/main.py:781
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Configurando el tema de Plymouth"

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Instalando data."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "Configura los servicios de OpenRC"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""

#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "No se puede modificar el servicio."

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "El nivel de ejecución del objetivo no existe"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "El servicio objetivo no existe"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd units"
msgstr ""

#: src/modules/services-systemd/main.py:64
msgid "Cannot modify unit"
msgstr ""

#: src/modules/services-systemd/main.py:65
msgid ""
"<code>systemctl {_action!s}</code> call in chroot returned error code "
"{_exit_code!s}."
msgstr ""

#: src/modules/services-systemd/main.py:66
msgid "Cannot {_action!s} systemd unit <code>{_name!s}</code>."
msgstr ""

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Llenando sistema de archivos."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "rsync falló con código de error {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr "Desempaquetando imagen {}/{}, archivo {}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Iniciando a desempaquetar {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "Se falló en la desempaquetización de la imagen \"{}\""

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "No existe punto de montaje para la partición raíz"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr ""

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Mal punto de montaje para la partición raíz"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr ""

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr ""

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr ""

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr ""
"Tu kernel actual no tiene soporte para el sistema de archivos para \"{}\" "
"({})"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "El sistema de archivos fuente \"{}\" no existe"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "El destino \"{}\" en el sistema objetivo no es un directorio"

#: src/modules/zfshostid/main.py:27
msgid "Copying zfs generated hostid."
msgstr ""
